src/day-1-toy/scope.py

```diff
@@ -4,10 +4,11 @@
 # When you use a variable in a function, it's local in scope to the function.
 x = 12
 
-def changeX():
-    x = 99
-
-changeX()
+def changeX(x):
+    global x
+    return x
+    # print(x)
+changeX(x)
```

Comment:
Seem to be making the right steps in the right location here, but let me know if you need additional help with this one. Don't forget this resource too: https://www.programiz.com/python-programming/global-local-nonlocal-variables